requirements.txt

```diff
@@ -71,7 +71,6 @@ pyzmq==25.0.0
 redis==4.5.1
 regex==2022.10.31
 requests==2.28.2
-rich==13.3.1
```

Comment:
Let's still keep the dependency in here, since rich is used in most of the example notebooks.